optimum/commands/export/openvino.py

```diff
@@ -68,8 +68,22 @@ def parse_args_openvino(parser: "ArgumentParser"):
             "This is needed by some models, for some tasks. If not provided, will attempt to use the tokenizer to guess it."
         ),
     )
-    optional_group.add_argument("--fp16", action="store_true", help="Compress weights to fp16"),
-    optional_group.add_argument("--int8", action="store_true", help="Compress weights to int8"),
+    optional_group.add_argument(
+        "-c",
+        "--compress-weights",
+        type=str,
+        choices=["f16", "i8", "i4_sym_g128", "i4_asym_g128", "i4_sym_g64", "i4_asym_g64"],
+        default=None,
```

Comment:
Can we have a way to not compress models at all and keep them in FP32? This is sometimes needed when evaluating models and comparing speedup/accuracy for example.

Reply:
Done